### DISCUSSION TITLE
Filename substitution filter?

### DISCUSSION THREAD

**Discussion by Shautora** [WinMerge/winmerge#2102] [Q&A]
I think that's what I'm looking for?

Here's my issue:  I'm trying to compare two large sets of files; in one, the files are prefixed with `00_`, and in the other, they're prefixed with `zpp_mod_`.  How can I get the folder comparison to view these two prefixes as being the same?

**Comment by sdottaka** [comment]
Unfortunately, WinMerge does not have the ability to ignore part of a folder or file name when comparing folders.
